### PR TITLE
feat(GlobalHeader): Global nav next gen header

### PR DESCRIPTION
**Pull request by katiezutter**
### Overview

Adds GlobalHeader component as well as AppHeader and its associated components. Does not yet address mobile version of header.

Free:
Screen Shot 2021-01-06 at 12 13 48 PM
Pro:
Screen Shot 2021-01-06 at 12 15 13 PM
Anonymous:
Screen Shot 2021-01-06 at 12 15 26 PM

Result of pairing with @saghdaey!

Implementation of suggested solution in RFC: https://www.notion.so/codecademy/Share-Global-Header-Between-Monolith-Static-Sites-afa63d25a43943738479f7015b92f1f0#826e13daaaba48a6a9e37877a91140dd

Here's the TLDR overview:

The consumer (monolith, static-sites) can pass in a `variant` prop (`anon`, `free`, `pro`) to `GlobalHeader` as well as a tracking function and render functions for search, notifications, and profile. `GlobalHeader` looks at the variant type and determines what items to pass to `AppHeader`, which takes in those item objects and maps them to actual components.

Check out the storybook to see examples of `GlobalHeader` variations! https://5ff603e72795c20f9a4cd101--gamut-preview.netlify.app/storybook/?path=/docs/labs-experimental-molecules-globalheader--global-header

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/7PNKwpTpXjRrPjcCRSu215/Global-Nav?node-id=897%3A14
- [x] Related to JIRA ticket: [EGG-539](https://codecademy.atlassian.net/browse/EGG-539), [EGG-540](https://codecademy.atlassian.net/jira/software/projects/EGG/boards/142?selectedIssue=EGG-540), [EN-350](https://codecademy.atlassian.net/browse/EN-350)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### Notes
* Work for adding the mobile menu and creating a separate, slimmed-down AnonymousGlobalHeader component will happen in future tickets.
* We also incorporated a small change requested by Puneet adding a "For Enterprise" link to the header. Ticket here: https://codecademy.atlassian.net/browse/EN-350

### To-Dos
Although we have a few to-dos left, we wanted to get feedback on what we currently have here before going further.

* ~~We have a test that is failing due to an invalid hook call: https://app.circleci.com/pipelines/github/Codecademy/client-modules/8829/workflows/e2e259dc-1e15-4214-9c8f-ed7a0cef9008/jobs/52340 - we're pretty stuck on how to solve this issue so please let us know if you have suggestions!~~
* Add unit tests
* ~~Change ButtonDeprecated components over to the new Gamut Buttons~~
* ~~Add tracking to the "Upgrade to Pro" button - checking with Natalia on this since it may have been missed in Global Nav Milestone I~~ Added `topnav_pro_trial` target
* ~~Deal with Redux First Routing considerations~~

*we know it's failing because we put the GlobalHeader component in a div with in line styling (so we could mimmic if the consumer rendered the global header on a page w/ a background color)- but we're not sure why that breaks the storybook-8-test, and how to get around it.

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.1e3ce5.0<br/>@codecademy/styleguide@21.7.1-alpha.1e3ce5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff37841d159f401294aa5b2--gamut-preview.netlify.app>https://5ff37841d159f401294aa5b2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff37841d159f401294aa5b2>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.16689d.0<br/>@codecademy/styleguide@21.7.1-alpha.16689d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff3851b34460f0ec3d390ae--gamut-preview.netlify.app>https://5ff3851b34460f0ec3d390ae--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff3851b34460f0ec3d390ae>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.bd2680.0<br/>@codecademy/styleguide@21.7.1-alpha.bd2680.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff38924ae41ae18b953034e--gamut-preview.netlify.app>https://5ff38924ae41ae18b953034e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff38924ae41ae18b953034e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.9a154f.0<br/>@codecademy/styleguide@21.7.1-alpha.9a154f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff38eaa65fa0b16bb6670a8--gamut-preview.netlify.app>https://5ff38eaa65fa0b16bb6670a8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff38eaa65fa0b16bb6670a8>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.ac0497.0<br/>@codecademy/styleguide@21.7.1-alpha.ac0497.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff3a12f1017263266af67be--gamut-preview.netlify.app>https://5ff3a12f1017263266af67be--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff3a12f1017263266af67be>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.eafbf2.0<br/>@codecademy/styleguide@21.7.1-alpha.eafbf2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff4a60038a80e37677d0380--gamut-preview.netlify.app>https://5ff4a60038a80e37677d0380--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff4a60038a80e37677d0380>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.2e6601.0<br/>@codecademy/styleguide@21.7.1-alpha.2e6601.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.dc9299.0<br/>@codecademy/styleguide@21.7.1-alpha.dc9299.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff4ac4c2a44ab4c5ddbfaa3--gamut-preview.netlify.app>https://5ff4ac4c2a44ab4c5ddbfaa3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff4ac4c2a44ab4c5ddbfaa3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.e03792.0<br/>@codecademy/styleguide@21.7.1-alpha.e03792.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff4bdebf82f9100bd0a98a1--gamut-preview.netlify.app>https://5ff4bdebf82f9100bd0a98a1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff4bdebf82f9100bd0a98a1>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.18adad.0<br/>@codecademy/styleguide@21.7.1-alpha.18adad.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.daec16.0<br/>@codecademy/styleguide@21.7.1-alpha.daec16.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff4c438dcc42900b5849cfd--gamut-preview.netlify.app>https://5ff4c438dcc42900b5849cfd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff4c438dcc42900b5849cfd>Deploy Logs</a></p>

File: packages/gamut-labs/src/experimental/GlobalHeader/AppHeaderItems.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    popover: renderProfile,
+    type: 'render-popover',
+  };
+};
+
+export const upgradeToPro: AppHeaderFillButton = {
+  id: 'upgrade-to-pro',
+  text: 'Upgrade to Pro',
+  href: '/pro/membership',
+  target: 'TODO', // TODO
```
Comment (saghdaey) [inline packages/gamut-labs/src/experimental/GlobalHeader/AppHeaderItems.tsx:181]: we noticed this button doesn't currently have tracking; checking in with Natalia + Richie about adding

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.3df450.0<br/>@codecademy/styleguide@21.7.1-alpha.3df450.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.edda22.0<br/>@codecademy/styleguide@21.7.1-alpha.edda22.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.c8e785.0<br/>@codecademy/styleguide@21.7.1-alpha.c8e785.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff5ff68d6f68110ac57c830--gamut-preview.netlify.app>https://5ff5ff68d6f68110ac57c830--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff5ff68d6f68110ac57c830>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.4-alpha.0c2c44.0<br/>@codecademy/styleguide@21.7.1-alpha.0c2c44.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff603e72795c20f9a4cd101--gamut-preview.netlify.app>https://5ff603e72795c20f9a4cd101--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff603e72795c20f9a4cd101>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.5-alpha.7c9c36.0<br/>@codecademy/styleguide@21.7.2-alpha.7c9c36.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff609d335901d1df396fd67--gamut-preview.netlify.app>https://5ff609d335901d1df396fd67--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff609d335901d1df396fd67>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.5-alpha.d7e54a.0<br/>@codecademy/styleguide@21.7.2-alpha.d7e54a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff60f703095f02bac3afc42--gamut-preview.netlify.app>https://5ff60f703095f02bac3afc42--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff60f703095f02bac3afc42>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.5-alpha.160522.0<br/>@codecademy/styleguide@21.7.2-alpha.160522.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff76b7418649e00b4e9dc8f--gamut-preview.netlify.app>https://5ff76b7418649e00b4e9dc8f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff76b7418649e00b4e9dc8f>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.5-alpha.c35721.0<br/>@codecademy/styleguide@21.7.2-alpha.c35721.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff7718caf8f1000d0918701--gamut-preview.netlify.app>https://5ff7718caf8f1000d0918701--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff7718caf8f1000d0918701>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.6.5-alpha.adc10d.0<br/>@codecademy/styleguide@21.7.2-alpha.adc10d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ff774a9dce0ea0c16b24d35--gamut-preview.netlify.app>https://5ff774a9dce0ea0c16b24d35--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ff774a9dce0ea0c16b24d35>Deploy Logs</a></p>

**Comment by saghdaey** [comment]
Most recently   changed the trackUserClick(target) to
onClick: (event: React.MouseEvent, trackingTarget: string) => void;

to which Aaron responded suggesting we pass the entire item object as the second arg since we may want to do routing or other things based on info that isn’t on the tracking target /  handle multiple side effects

File: packages/gamut-labs/src/experimental/AppHeader/index.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+import { AppHeaderLinkElement } from '../AppHeaderElements/AppHeaderLinkElement';
+import { AppHeaderLogoElement } from '../AppHeaderElements/AppHeaderLogoElement';
+import { AppHeaderTab } from '../AppHeaderElements/AppHeaderTab';
+import styles from './styles.scss';
+import { AppHeaderItem } from './types';
+
+export type AppHeaderProps = {
+  items: AppHeaderItemsProp;
+  className?: string;
+  onClick: (event: React.MouseEvent, trackingTarget: string) => void;
```
Comment (codecaaron) [inline packages/gamut-labs/src/experimental/AppHeader/index.tsx:16]: Do we want to make these handlers a bit more flexible in this iteration?  

I thought we landed on something like this:
```tsx
type HeaderItemShape = AppHeaderLink | AppHeaderLogo;
type HeaderClickHandler = (event: React.MouseEvent, item: HeaderItemShape) => void;
```
That way we can use anything on the config to handle side effects like routing as well as tracking.

Forgive me if I misremembered :)

**Review by codecaaron — COMMENTED** [review]
This is looking clean!

Would y'all be up for pairing on using emotion for the styling?

File: packages/gamut-labs/src/experimental/AppHeader/index.tsx (hunk 2 above)
Comment (saghdaey) [reply packages/gamut-labs/src/experimental/AppHeader/index.tsx:16]: hi! yes for the click handler, i wrote your feedback re passing the entire object in a comment above but haven't tackled it yet!

Comment (codecaaron) [reply packages/gamut-labs/src/experimental/AppHeader/index.tsx:16]: Awesome :D I saw your comment after I posted, sorry for my confusion!

File: packages/gamut-labs/src/experimental/AppHeaderElements/AppHeaderDropdown/styles.scss [hunk 3]
```diff
@@ -0,0 +1,80 @@
+@import "~@codecademy/gamut-styles/utils";
+
+.dropdown {
+  background-color: $color-white;
+  line-height: normal;
+  @include font-smoothing(off);
+  padding: 0.7rem 0;
+}
```
[the remark this answers is not in the document]
Comment (katiezutter) [reply packages/gamut-labs/src/experimental/AppHeaderElements/AppHeaderDropdown/styles.scss:8]: That would be fantastic! Been working on removing the deprecated buttons and using emotion, but I got stuck here.

File: packages/gamut-labs/src/experimental/AppHeader/index.tsx (hunk 2 above)
Comment (saghdaey) [reply packages/gamut-labs/src/experimental/AppHeader/index.tsx:14]: np!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.7.3-alpha.5e5256.0<br/>@codecademy/styleguide@21.9.1-alpha.5e5256.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ffdc90326f64331a11765d8--gamut-preview.netlify.app>https://5ffdc90326f64331a11765d8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ffdc90326f64331a11765d8>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.7.3-alpha.84a65b.0<br/>@codecademy/styleguide@21.9.1-alpha.84a65b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ffdebc9f4179600d05242fd--gamut-preview.netlify.app>https://5ffdebc9f4179600d05242fd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ffdebc9f4179600d05242fd>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.7.3-alpha.ed010e.0<br/>@codecademy/styleguide@21.9.1-alpha.ed010e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ffe1321b675d400961869ba--gamut-preview.netlify.app>https://5ffe1321b675d400961869ba--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ffe1321b675d400961869ba>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.7.3-alpha.c53c8d.0<br/>@codecademy/styleguide@21.9.1-alpha.c53c8d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ffe15bc9a652f00f9486529--gamut-preview.netlify.app>https://5ffe15bc9a652f00f9486529--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ffe15bc9a652f00f9486529>Deploy Logs</a></p>

File: packages/gamut-labs/src/experimental/GlobalHeader/index.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  anonHeaderItems,
+  freeHeaderItems,
+  proHeaderItems,
+} from './AppHeaderVariants';
+import styles from './styles.scss';
+
+export type HeaderClickHandler = (
+  event: React.MouseEvent,
+  item: AppHeaderItem
+) => void;
```
Comment (saghdaey) [inline packages/gamut-labs/src/experimental/GlobalHeader/index.tsx:15]: @codecaaron implemented your feedback here to add type for the onClick

File: packages/gamut-labs/src/experimental/AppHeader/types.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
+import { ReactElement } from 'react';
+
+export type AppHeaderItem =
+  | AppHeaderLogo
+  | AppHeaderLink
+  | AppHeaderTextButton
+  | AppHeaderFillButton
+  | AppHeaderPopover
+  | AppHeaderRenderPopover;
+
```
Comment (saghdaey) [inline packages/gamut-labs/src/experimental/AppHeader/types.tsx:10]: union type for app header items

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.7.3-alpha.343a67.0<br/>@codecademy/styleguide@21.9.1-alpha.343a67.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ffe2fbc3c5e9317131f9b14--gamut-preview.netlify.app>https://5ffe2fbc3c5e9317131f9b14--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5ffe2fbc3c5e9317131f9b14>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.7.3-alpha.9585b9.0<br/>@codecademy/styleguide@21.9.1-alpha.9585b9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fff15c13c5e93409f1f9926--gamut-preview.netlify.app>https://5fff15c13c5e93409f1f9926--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fff15c13c5e93409f1f9926>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.7.4-alpha.7bd246.0<br/>@codecademy/styleguide@21.9.2-alpha.7bd246.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fff196373761838262b2f81--gamut-preview.netlify.app>https://5fff196373761838262b2f81--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fff196373761838262b2f81>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.7.5-alpha.1f9172.0<br/>@codecademy/styleguide@21.10.1-alpha.1f9172.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fff27bcb5b90a08979d88f5--gamut-preview.netlify.app>https://5fff27bcb5b90a08979d88f5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fff27bcb5b90a08979d88f5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.7.5-alpha.7dec9b.0<br/>@codecademy/styleguide@21.10.1-alpha.7dec9b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fff2ea0b3aa71161fa3f2e7--gamut-preview.netlify.app>https://5fff2ea0b3aa71161fa3f2e7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fff2ea0b3aa71161fa3f2e7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.7.5-alpha.aa01f5.0<br/>@codecademy/styleguide@21.10.1-alpha.aa01f5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fffea84bfe349697d6a282e--gamut-preview.netlify.app>https://5fffea84bfe349697d6a282e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fffea84bfe349697d6a282e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.7.5-alpha.252528.0<br/>@codecademy/styleguide@21.10.1-alpha.252528.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6000b5efd65f4f621df01db4--gamut-preview.netlify.app>https://6000b5efd65f4f621df01db4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6000b5efd65f4f621df01db4>Deploy Logs</a></p>

**Pull request closed without merging**